MMR.Randomizer/Utils/RomUtils.cs

```diff
@@ -337,14 +337,29 @@ public static byte[] BuildROM()
                     continue;
                 }
                 RomData.MMFileList[i].Cmp_Addr = ROMAddr;
-                int file_len = RomData.MMFileList[i].Data.Length;
+                int FileLength = RomData.MMFileList[i].Data.Length;
```

Comment:
Local variable names should be in lowerCamelCase.

Reply:
changed to lowerCamelCase